### PR TITLE
Chapter/chapter12/cat.asciidoc 翻译完成

### PR DESCRIPTION
**Pull request by zhuan77241**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: docs/reference/cat.asciidoc [hunk 1]
```diff
@@ -44,18 +41,17 @@ u_n93zwxThWHi1PDBJAGAg 127.0.0.1 127.0.0.1 u_n93zw
 
 [float]
 [[help]]
-=== Help
+=== 帮助
 
-Each of the commands accepts a query string parameter `help` which will
-output its available columns. For example:
+每个命令都接受一个字符串参数 `help`，它将会输出其可用列。 例如：
```
Comment (autumnbz) [inline docs/reference/cat.asciidoc]: `help`，中文逗号前需要空格，否则预览的时候格式会乱

File: docs/reference/cat/fielddata.asciidoc [hunk 2]
```diff
@@ -1,8 +1,7 @@
 [[cat-fielddata]]
 == cat fielddata
 
-`fielddata` shows how much heap memory is currently being used by fielddata
-on every data node in the cluster.
+`fielddata` 显示集群中每个数据节点上fielddata当前正在使用多少堆内存。
```
Comment (Littlesqx) [inline docs/reference/cat/fielddata.asciidoc]: 显示了集群中每个数据节点上被 fielddata 正在使用的堆内存大小。

File: docs/reference/cat/health.asciidoc [hunk 3]
```diff
@@ -52,10 +50,9 @@ across nodes:
 --------------------------------------------------
 // NOTCONSOLE
 
-A less obvious use is to track recovery of a large cluster over
-time. With enough shards, starting a cluster, or even recovering after
-losing a node, can take time (depending on your network & disk). A way
-to track its progress is by using this command in a delayed loop:
+一种少见的用途是随着时间追踪大型集群的恢复。
+有了足够的碎片后，在启动集群，甚至在丢失节点后恢复，需要耗费时间（取决于您的网络和磁盘）。
```
Comment (Littlesqx) [inline docs/reference/cat/health.asciidoc]: 你

File: docs/reference/cat/health.asciidoc [hunk 4]
```diff
@@ -68,19 +65,17 @@ to track its progress is by using this command in a delayed loop:
...
+如果这个数字保持不变，我们则会认为这是一个问题。
 
 [float]
 [[timestamp]]
 === Why the timestamp?
 
-You typically are using the `health` command when a cluster is
-malfunctioning.  During this period, it's extremely important to
-correlate activities across log files, alerting systems, etc.
+当群集发生故障时，您通常使用 `health` 命令。
```
Comment (Littlesqx) [inline docs/reference/cat/health.asciidoc]: 集群，你

File: docs/reference/cat/health.asciidoc [hunk 5]
```diff
@@ -68,19 +65,17 @@ to track its progress is by using this command in a delayed loop:
...
+在此期间，关联活动的访问日志文件，系统报警等非常重要。
 
-There are two outputs.  The `HH:MM:SS` output is simply for quick
-human consumption.  The epoch time retains more information, including
-date, and is machine sortable if your recovery spans days.
+这里有两个输出：
+ `HH:MM:SS` 输出是简单的，方便人们识别。
+ epoch 时间保留更多信息，包括日期，如果您的恢复时间跨度在几天内，则可以通过机器进行排序。
```
Comment (Littlesqx) [inline docs/reference/cat/health.asciidoc]: 你

File: docs/reference/cat/indices.asciidoc [hunk 6]
```diff
@@ -23,31 +22,27 @@ green  open   twitter2 nYFWZEO7TUiOjLQXBaYJpA   5   0          0            0
...
 
 *Notes:*
 
-1. As the number of documents and deleted documents shown in this are at the lucene level,
-it includes all the hidden documents (e.g. from nested documents) as well.
+1. 由于此处显示的文档和已删除文档的数量均为 lucene 级别，因此它还包含所有隐藏文档（例如嵌套文档）。
 
-2. To get actual count of documents at the Elasticsearch level, the recommended way
-is to use either the <<cat-count>> or the <<search-count>>
+2. 要在Elasticsearch级别获得实际的文档数量，推荐的方法是使用 << cat-count >> 或者 << search-count >>
```
Comment (Littlesqx) [inline docs/reference/cat/indices.asciidoc]: 空格

File: docs/reference/cat/nodes.asciidoc [hunk 7]
```diff
@@ -20,35 +20,29 @@ ip        heap.percent ram.percent cpu load_1m load_5m load_15m node.role master
 // TESTRESPONSE[s/65          99  42/\\d+ \\d+ \\d+/]
 // TESTRESPONSE[s/[*]/[*]/ s/mJw06l1/.+/ _cat]
 
-The first few columns (`ip`, `heap.percent`, `ram.percent`, `cpu`, `load_*`) tell
-you where your nodes live and give a quick picture of performance stats.
+前几列（ `ip`, `heap.percent`, `ram.percent`, `cpu`, `load_*` ）告诉你你的节点在哪里，以便快速了解性能统计信息。
```
Comment (Littlesqx) [inline docs/reference/cat/nodes.asciidoc]: 去掉你的

File: docs/reference/cat/nodes.asciidoc [hunk 8]
```diff
@@ -20,35 +20,29 @@ ip        heap.percent ram.percent cpu load_1m load_5m load_15m node.role master
...
 
-The last (`node.role`, `master`, and `name`) columns provide ancillary
-information that can often be useful when looking at the cluster as a whole,
-particularly large ones.  How many master-eligible nodes do I have?
+最后几列（ `node.role` ， `master` 和 `name` ）提供了辅助信息，这在查看整个群集时通常很有用，特别是大型群集。
```
Comment (Littlesqx) [inline docs/reference/cat/nodes.asciidoc]: 集群

File: docs/reference/cat/segments.asciidoc [hunk 9]
```diff
@@ -22,52 +21,34 @@ test1 3     p      127.0.0.1 _0               0          1            0  3kb
...
 // TESTRESPONSE[s/3kb/\\d+(\\.\\d+)?[mk]?b/ s/2042/\\d+/ _cat]
 
-The output shows information about index names and shard numbers in the first
-two columns.
+输出显示了前两列中的索引名称和分片数字信息。
 
-If you only want to get information about segments in one particular index,
-you can add the index name in the URL, for example `/_cat/segments/test`. Also,
-several indexes can be queried like `/_cat/segments/test,test1`
+如果您只想获取某个特定索引的 segments 信息，可以在URL中添加索引名称，例如 `/_cat/segments/test` 。
```
Comment (Littlesqx) [inline docs/reference/cat/segments.asciidoc]: 你  空格

File: docs/reference/cat/segments.asciidoc [hunk 10]
```diff
@@ -22,52 +21,34 @@ test1 3     p      127.0.0.1 _0               0          1            0  3kb
 --------------------------------------------------
 // TESTRESPONSE[s/3kb/\\d+(\\.\\d+)?[mk]?b/ s/2042/\\d+/ _cat]
 
-The output shows information about index names and shard numbers in the first
-two columns.
+输出显示了前两列中的索引名称和分片数字信息。
```
Comment (Littlesqx) [inline docs/reference/cat/segments.asciidoc]: 在前两列的输出信息中展示了索引名和分片数量。

File: docs/reference/cat/segments.asciidoc [hunk 11]
```diff
@@ -22,52 +21,34 @@ test1 3     p      127.0.0.1 _0               0          1            0  3kb
...
 
 
-The following columns provide additional monitoring information:
+以下各列提供了其他监控信息：
 
-prirep::        Whether this segment belongs to a primary or replica shard.
+prirep::        此 segment 是属于主分片还是副本分片。
 
-ip::            The ip address of the segment's shard.
+ip::            segment 所在分片的IP地址。
```
Comment (Littlesqx) [inline docs/reference/cat/segments.asciidoc]: 空格

File: docs/reference/cat/shards.asciidoc [hunk 12]
```diff
@@ -29,9 +27,8 @@ twitter 0 p STARTED 3014 31.1mb 192.168.56.10 H5dfFeA
 [[index-pattern]]
 === Index pattern
 
-If you have many shards, you may wish to limit which indices show up
-in the output.  You can always do this with `grep`, but you can save
-some bandwidth by supplying an index pattern to the end.
+如果您有许多分片，您希望限制输出中显示的索引。
```
Comment (Littlesqx) [inline docs/reference/cat/shards.asciidoc]: 你

File: docs/reference/cat/shards.asciidoc [hunk 13]
```diff
@@ -56,8 +53,8 @@ twitter 0 p STARTED 3014 31.1mb 192.168.56.10 H5dfFeA
 [[relocation]]
 === Relocation
 
-Let's say you've checked your health and you see a relocating
-shards.  Where are they from and where are they going?
+假设您检查了集群的健康状况，并且看到了重新分配中的分片。
```
Comment (Littlesqx) [inline docs/reference/cat/shards.asciidoc]: 你

File: docs/reference/cat/shards.asciidoc [hunk 14]
```diff
@@ -97,11 +93,10 @@ twitter 0 r INITIALIZING    0 14.3mb 192.168.56.30 bGG90GE
...
 // TESTRESPONSE[_cat]
 
-If a shard cannot be assigned, for example you've overallocated the
-number of replicas for the number of nodes in the cluster, the shard
-will remain `UNASSIGNED` with the <<reason-unassigned,reason code>> `ALLOCATION_FAILED`.
+如果无法分配分片，例如，您已经为集群中分配的副本数量超过了节点数量，
+那么分片将保持 `UNASSIGNED` 因为 <<reason-unassigned,reason code>> `ALLOCATION_FAILED` 。
 
-You can use the shards API to find out that reason.
+您可以使用 shards API 找出原因。
```
Comment (Littlesqx) [inline docs/reference/cat/shards.asciidoc]: 你

File: docs/reference/cat/thread_pool.asciidoc [hunk 15]
```diff
@@ -99,45 +99,43 @@ id                     name    active rejected completed
...
 |=======================================================================
 
 [float]
 === Other Fields
 
-In addition to details about each thread pool, it is also convenient to get an
-understanding of where those thread pools reside. As such, you can request
-other details like the `ip` of the responding node(s).
+除了每个线程池的详细信息之外，了解线程池驻留的位置也很方便。
+因此，您可以请求其他详细信息，例如响应节点的 `ip` 。
```
Comment (Littlesqx) [inline docs/reference/cat/thread_pool.asciidoc]: 你

**Pull request merged**